### ISSUE TITLE
Документация/API 6

### ISSUE THREAD

**Issue by ilysharusher** [irazasyed/telegram-bot-sdk#958]
Здравствуйте, в скором времени появиться документация к актуальным версиям библиотеки? И стоит ли ждать обновлений параллельно версиям самих API?

**Comment by alies-dev** [comment]
@ilysharusher 
this is not a russian-speaking repo, sorry.

duplicate of the https://github.com/irazasyed/telegram-bot-sdk/issues/603

**Comment by ilysharusher** [comment]
Hello, will the documentation for the current versions of the library appear soon? And is it worth waiting for updates in parallel with API versions?

**Comment by alies-dev** [comment]
@ilysharusher 
I assume SDK v3 is not a priority as the community works on v4: https://github.com/telegram-bot-sdk/telegram-bot-sdk (that also doesn't have docs). The only well-documented version is v2
